### ISSUE TITLE
ImportError: No module named 'jwt'

### ISSUE THREAD

**Issue by armicron** [capless/warrant#6]
```
Traceback (most recent call last):
  File "interactive.py", line 1, in <module>
    from warrant import Cognito
  File "/home/zzz/coding/python/cognito/warrant/warrant/__init__.py", line 4, in <module>
    import jwt
ImportError: No module named 'jwt'
```

**Comment by bjinwright** [comment]
Good catch pyjwt should be in the requirements.txt

**Comment by bjinwright** [comment]
Fixed in 442402e2011c8f562a98c767dbb050e3034f23af
